1-js/03-code-quality/06-polyfills/article.md

```diff
@@ -1,57 +1,56 @@
 
 # Polyfills
 
-The JavaScript language steadily evolves. New proposals to the language appear regularly, they are analyzed and, if considered worthy, are appended to the list at <https://tc39.github.io/ecma262/> and then progress to the [specification](http://www.ecma-international.org/publications/standards/Ecma-262.htm).
+A linguagem JavaScript evolui constantemente. Novas propostas à linguagem aparecem regularmente, onde elas são analisadas e, se consideradas que valem a pena, anexadas a lista em <https://tc39.github.io/ecma262/>, onde depois avançam para a [especificação](http://www.ecma-international.org/publications/standards/Ecma-262.htm).
```

Comment:
Hi,

I do not know what you've done, since it appears that you changed where it was good and left where I asked for a change.
- Would you please revert that change at '... propostas à linguagem ...' , and consider my comment above?
Thanks.